core/modules/node/css/node.admin.css

```diff
@@ -10,3 +9,10 @@
 .revision-current {
   background: #ffc;
 }
+
+/**
+ * Allow permissions matrix to scroll on "Add Content Type" form.
+ */
+#edit-permissions > .fieldset-wrapper > .form-item {
+  overflow-x: scroll;
```

Comment:
If setting overflow-x to scroll, the scrollbar below will always be there, even if not necessary.
I belief, this should be "auto", so the scrollbar only appears when needed.

Reply:
Indeed, thanks! Now fixed.